data/transactions/logic/assembler.go

```diff
@@ -425,7 +425,7 @@ func assembleInt(ops *OpStream, spec *OpSpec, args []string) error {
 }
 
 // Explicit invocation of const lookup and push
-func assembleIntC(ops *OpStream, spec *OpSpec, args []string) error {
+func asmIntC(ops *OpStream, spec *OpSpec, args []string) error {
```

Comment:
@jannotti Minor:  Is there a by-design reason why https://github.com/algorand/go-algorand/pull/3612/commits/85db0ac73fe00d0c48a728a08209547e90220dca _omits_ renaming functions used in `keywords` from `assembler.go` (e.g. `assembleInt`)?

I might be missing context.  Judging from the commit diff and name, I'm assuming the intent is to universally change naming within AVM.

Reply:
I _only_ renamed all of the functions that perform assembly (and the name of their type).